spring-ai-core/src/main/java/org/springframework/ai/chat/client/observation/ChatClientObservationContext.java

```diff
@@ -0,0 +1,63 @@
+/*
+* Copyright 2024 - 2024 the original author or authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* https://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.springframework.ai.chat.client.observation;
+
+import org.springframework.ai.chat.client.DefaultChatClient.DefaultChatClientRequestSpec;
+import org.springframework.ai.observation.AiOperationMetadata;
+
+import io.micrometer.observation.Observation;
+
+/**
+ * @author Christian Tzolov
+ * @since 1.0.0
+ */
+
+public class ChatClientObservationContext extends Observation.Context {
+
+	private final boolean stream;
+
+	private final String format;
+
+	private final DefaultChatClientRequestSpec request;
+
+	private final AiOperationMetadata operationMetadata;
+
+	public ChatClientObservationContext(DefaultChatClientRequestSpec requestSpec, AiOperationMetadata operationMetadata,
```

Comment:
Since this context is used specifically for ChatClient observations, we could simplify the constructor by removing AiOperationMetadata, building an instance inside the constructor with fixed values "spring_ai" and "framework". I simplified in a similar way the constructors for the model contexts (e.g.: https://github.com/spring-projects/spring-ai/blob/bf84d5945e9499d01a9deff9435b2e772804afab/spring-ai-core/src/main/java/org/springframework/ai/chat/observation/ChatModelObservationContext.java#L38)